styles.css

```diff
@@ -343,3 +346,276 @@ footer {
+    line-height: 82px;
+  }
+
+  .introText > .lh32 {
+    font-weight: 500;
+    font-size: 32px;
+    line-height: 44px;
+  }
+
+  .introText > .lh24 {
+    font-size: 20px;
+    line-height: 32px;
+  }
+
+  .socialIcons > ul {
+    padding-top: 87px;
+    display: flex;
+    flex-direction: column;
+    gap: 20px;
+  }
+
+  #works {
+    display: flex;
+    padding: 135px 9.9vw 130px 9.7vw;
+    height: auto;
+  }
+
+  .worksGrid {
+    width: 78.6vw;
+    display: grid;
+    grid-template-rows: 52px 114px 448px 24px 435px 37px 435px;
+    grid-template-columns: 25.6vw 1.6vw 25.6vw 1.6vw 25.6vw;
+    row-gap: 0;
+  }
+
+  .grid1 {
+    grid-row: 1/2;
+    grid-column: 1/8;
+    flex-direction: row;
+  }
+
+  .grid1 > h1 {
+    margin-right: 24px;
+  }
+
+  .grid2 {
+    grid-row: 3/4;
+    grid-column: 1/8;
+    flex-direction: row;
+  }
+
+  .grid3 > h1 { display: none; }
+  .grid3 > p { display: none; }
+
+  .aboutMe > h1 { line-height: 81.5px; }
+
+  .aboutMe > p {
+    font-size: 20px;
+    line-height: 32px;
+  }
+
+  .grid2 > div > h1 {
+    font-size: 40px;
+    line-height: 52px;
+  }
+
+  .grid2 > div > p {
+    font-size: 20px;
+    line-height: 28px;
+  }
+
+  .grid3 {
+    grid-row: 5/6;
+    grid-column: 1/2;
+    background-image: url("img/deskGrid3.jpg");
+    background-repeat: no-repeat;
+    background-position: center;
+    justify-content: end;
+  }
+
+  .grid3 > ul { display: none; }
+  .grid3 > button { width: 25.6vw; }
+
+  .project2 { gap: 0; }
+
+  .grid4 {
+    grid-row: 5/6;
+    grid-column: 3/4;
+    background-image: url("img/deskGrid4.jpg");
+    background-repeat: no-repeat;
+    background-position: center;
+    gap: 24px;
+    padding-top: 156px;
+  }
+  .grid4 > button { display: none; }
+
+  .grid5 {
+    grid-row: 5/6;
+    grid-column: 5/6;
+    background-image: url("img/deskGrid5.jpg");
+    background-repeat: no-repeat;
+    background-position: center;
+    gap: 24px;
+    padding-top: 200px;
+  }
+  .grid5 > button { display: none; }
+
+  .grid6 {
+    grid-row: 7/8;
+    grid-column: 1/2;
+    gap: 24px;
+    padding-top: 156px;
+    background-image: url("img/deskGrid6.jpg");
+    background-repeat: no-repeat;
+    background-position: center;
+  }
+  .grid6 > button { display: none; }
+
+  .grid7 {
+    grid-row: 7/8;
+    grid-column: 3/4;
+    background-image: url("img/deskGrid4.jpg");
+    background-repeat: no-repeat;
+    background-position: center;
+    gap: 24px;
+    padding-top: 156px;
+  }
+  .grid7 > button { display: none; }
+
+  .grid8 {
+    grid-row: 7/8;
+    grid-column: 5/6;
+    background-image: url("img/deskGrid5.jpg");
+    background-repeat: no-repeat;
+    background-position: center;
+    gap: 24px;
+    padding-top: 200px;
+  }
+  .grid8 > button { display: none; }
+
+  #about {
+    background-image: url("img/deskAbout.png");
+    background-repeat: no-repeat;
+    background-position: 100% 8.3%;
+  }
+
+  .aboutMe { width: 39.3vw; }
+
+  .skills {
```

Comment:
Good work implementing the skills section, but you have to do some changes to fit the FIGMA design

- [ ] Kindly give each section more space between each list of items
- [ ] Kindly align the list of items to left on each section

![image](https://user-images.githubusercontent.com/93630700/206734353-d113eef7-ffd1-450c-a963-c9a941dec882.png)